lib/screens/chatscreen/chat_modules/chat_detail_page.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:shareacab/screens/chatscreen/chat_components/chat_bubble.dart';
+import 'package:shareacab/screens/chatscreen/chat_components/chat_detail_page_appbar.dart';
+import 'package:shareacab/screens/chatscreen/chat_models/chat_message.dart';
+import 'package:shareacab/screens/chatscreen/chat_models/send_menu_items.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+enum MessageType{
+  Sender,
+  Receiver,
+}
+
+
+class ChatDetailPage extends StatefulWidget{
+  @override
+  _ChatDetailPageState createState() => _ChatDetailPageState();
+}
+
+class _ChatDetailPageState extends State<ChatDetailPage> {
+  List<ChatMessage> chatMessage = [
+    ChatMessage(message: 'Hi John', type: MessageType.Receiver),
+    ChatMessage(message: 'Hope you are doin good', type: MessageType.Receiver),
+    ChatMessage(message: "Hello Jane, I'm good what about you", type: MessageType.Sender),
+    ChatMessage(message: "I'm fine, Working from home", type: MessageType.Receiver),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+  ];
+
+  List<SendMenuItems> menuItems = [
+    SendMenuItems(text: 'Photos & Videos', icons: Icons.image, color: Colors.amber),
+    SendMenuItems(text: 'Document', icons: Icons.insert_drive_file, color: Colors.blue),
+    SendMenuItems(text: 'Audio', icons: Icons.music_note, color: Colors.orange),
+    SendMenuItems(text: 'Location', icons: Icons.location_on, color: Colors.green),
+    SendMenuItems(text: 'Contact', icons: Icons.person, color: Colors.purple),
+  ];
+
+  void showModal(){
+    showModalBottomSheet(
+        context: context,
+        builder: (context){
+          return Container(
+            height: MediaQuery.of(context).size.height/2,
+            color: Color(0xff737373),
+            child: Container(
+              decoration: BoxDecoration(
+                color: Colors.white,
+                borderRadius: BorderRadius.only(topRight: Radius.circular(20),topLeft: Radius.circular(20)),
+              ),
+              child: Column(
```

Comment:
```
════════ (116) Exception caught by rendering library ═══════════════════════════════════════════════
A RenderFlex overflowed by 23 pixels on the bottom.
The relevant error-causing widget was: 
  Column file:///Users/vishal/Desktop/Coding%20B/Flutter/workspace/shareacab/lib/screens/chatscreen/chat_modules/chat_detail_page.dart:48:22
```
This is causing this error

<img width="311" alt="Screenshot 2020-05-15 at 7 27 59 PM" src="https://user-images.githubusercontent.com/31121102/82058551-84064480-96e2-11ea-8d10-cb4e3b47eaf4.png">